governance.md

```diff
@@ -0,0 +1,340 @@
+
+It is expected that the Council Members and BDFL will be employed at a wide
+range of companies, universities and non-profit organizations. Because of this,
+it is possible that Members will have conflict of interests. Such conflict of
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment, or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+-   An issue where the person privately gains an advantage from The Project
+    resources, but The Project has no gain or suffers a disadvantage.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused his/herself for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+### Private communications of the Council
+
+Unless specifically required, all Council discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
+and Community. The Council will have a private mailing list that will be used
+sparingly and only when a specific matter requires privacy. When private
+communications and decisions are needed, the Council will do its best to
+summarize those to the Community after eliding personal/private/sensitive
+information that should not be posted to the public internet.
+
+### Subcommittees
+
+The Council can create subcommittees that provide leadership and guidance for
+specific aspects of the project. Like the Council as a whole, subcommittees
+should conduct their business in an open and public manner unless privacy is
+specifically called for. Private subcommittee communications should happen on
+the main private mailing list of the Council unless specifically called for.
+
+The BDFL remains override authority over subcommittees, but should typically
+appoint a delegate who plays that role most of the time.
+
+### NumFOCUS Subcommittee
+
+The Council will maintain one narrowly focused subcommittee to manage its
+interactions with NumFOCUS.
+
+-   The NumFOCUS Subcommittee is comprised of 5 persons who manage project
+    funding that comes through NumFOCUS. It is expected that these funds will
+    be spent in a manner that is consistent with the non-profit mission of
+    NumFOCUS and the direction of the Project as determined by the full
+    Council.
+-   This Subcommittee shall NOT make decisions about the direction, scope or
+    technical direction of the Project.
+-   This Subcommittee will have 5 members, 4 of whom will be current Council
+    Members and 1 of whom will be external to the Steering Council. No more
```

Comment:
What is the purpose of the 1 person external to the Steering Council?

Reply:
I don't know

Reply:
If the purpose is unknown, may I suggest we drop this line?

Reply:
Looks like this was changed below. Marking as resolved. Please reopen if not.